osctiny/extensions/packages.py

```diff
@@ -186,7 +186,7 @@ def get_file(self, project, package, filename, meta=False, rev=None,
 
     # pylint: disable=too-many-arguments
     def download_file(self, project, package, filename, destdir, meta=False,
-                      overwrite=False, rev=None):
+                      overwrite=False, rev=None, expand=0):
```

Comment:
Just a little change to indicate that this should be considered as a boolean flag
```suggestion
                      overwrite=False, rev=None, expand=False):
```